_data/sites.json

```
@@ -3120,6 +3120,17 @@
         ]
     },
 
+    {
+        "name": "CV online",
+        "url": "https://cvonline.me/en/support/accounts/removing-account",
+        "difficulty": "hard",
+        "email": "hello@cvonline.me",
+        "notes": "Request account removal by sending an email at hello@cvonline.me from the same email address appearing on your CVs.",
```

```suggestion
        "notes": "Request account removal by sending an email to them from the same email address appearing on your CVs.",
```
Since there's a field for e-mail, we typically leave the notes without them to unclutter.